src/scss/_objects.stack.scss

```diff
@@ -0,0 +1,62 @@
+.grommet .stack {
+
+  &__label {
+    letter-spacing: normal;
+    margin-bottom: 0.25em;
+  }
+
+  &__headline {
+    margin-bottom: 0.125em;
+  }
+
+  &__paragraph {
+    &.grommetux-paragraph {
+      margin-top: 0.125em;
+      margin-bottom: 0.5em;
+    }
+  }
+
+  &__print {
+    &.grommetux-paragraph {
+      margin-bottom: 0.333em;
+    }
+  }
+
+  &__link {
+    margin-top: 0.75em;
+  }
+
+  &--small {
+    .stack__headline {
+      @include media-query(portable) {
+        font-size: 18px;
+      }
+    }
+  }
+
+  &--medium {
+    .stack__headline {
+      @include media-query(portable) {
+        font-size: 24px;
+      }
+    }
+  }
+
+  &--large {
+    .stack__headline {
+      @include media-query(portable) {
+        font-size: 36px;
+      }
+    }
+  }
+
+  &--xlarge {
+    .stack__headline {
+      font-size: 64px;
+      @include media-query(portable) {
+        font-size: 48px;
+      }
+    }
+  }
+
+}
```

Comment:
I believe lint usually complains if there's not a newline at the end of the file.